index.html

```diff
@@ -1,32 +1,131 @@
 <!DOCTYPE html>
+<link rel="stylesheet" type="text/css" href="/__assets/index.css" media="screen" />
 <html>
-<title>HackOregon API Service Catalog</title>
-<xmp theme="united" style="display:none;">
-
-### 2018 APIs
-
-* [The 2018 Disaster Resilience API](/disaster-resilience/) ([source](https://github.com/hackoregon/disaster-resilience-backend))
-
-* [The 2018 Housing Affordability API](/housing-affordability/) ([source](https://github.com/hackoregon/housing-2018))
-
-* [The 2018 Local Elections API](/local-elections/) ([source](https://github.com/hackoregon/elections-2018-backend))
-
-* [The 2018 Neighborhood Development API](/neighborhood-development/) ([source](https://github.com/hackoregon/neighborhoods-2018))
 
-* [The 2018 Transportation Systems API](/transportation-systems/) ([source](https://github.com/hackoregon/transportation-systems-backend-2018))
-
-### 2017 APIs
-
-* [The 2017 Budget API](/budget/) ([source](https://github.com/hackoregon/team-budget))
-
-* [The 2017 Emergency Response API](/emergency/) ([source](https://github.com/hackoregon/emergency-response-backend))
-
-* [The 2017 Homelessness API](/homeless/) ([source](https://github.com/hackoregon/teamhomelessness))
-
-* [The 2017 Housing API](/housing/) ([source](https://github.com/hackoregon/housing-backend))
+<title>HackOregon API Service Catalog</title>
 
-* [The 2017 Transportation API](/transport/) ([source](https://github.com/hackoregon/transportation-backend))
+<div class="collectionPageWrapper">
+	<div class="leftContainer">
+	  <div class="homeLink"><a href="http://civicplatform.org/"><img src="/__assets/civic-logo-animated.svg"/></a></div>
+	  <h1>Portland APIs</h1>
+	  <div class="subCopy">
+	    Portland APIs are built by teams of volunteers who are passionate about making public data open and accessible. The documentation and source code for each of these APIs is available on GitHub.
+	  </div>
+	</div>
+	<div class="sideListWrapper">
+	  <ul>
+	    <a href="/disaster-resilience/">
+	    	<li class="itemStyle">
+	    		<div class="teamTitleStyle">Hack Oregon 2018</div>
+	    		<h2>Disaster Resilience</h2>
+	    		<a href="https://github.com/hackoregon/disaster-resilience-backend">
+	    			<div class="github">
+	    				<img src="/__assets/github-icon.svg"/>
+	    			</div>
+	    		</a>
```

Comment:
I'm sure this works, but it's technically illegal to put an `a` in an `a`. It might be better to make this a two column flexbox.